### DISCUSSION TITLE
Organization-level required workflows don't respect bypassing rules

### DISCUSSION THREAD

**Discussion by crqrdotcom** [community/community#52037] [Actions]
### Select Topic Area

Bug

### Body

Hey there 👋🏽 - I'm not 100% sure this is a bug or is the intended behavior.

I have some workflows that have to push commits to repositories (with branch protection rules) before a release. To avoid the need for a PR/running checks I've added the GitHub App in the _Restrict who can dismiss pull request reviews_ and _Allow specified actors to bypass required pull requests_ sections.

After adding another workflow as required workflow at the organization-level, these commits stopped working with the message "Organization mandated required workflow checks need to run successfully to merge to this branch".

Is this the expected behavior? If so, what's the alternative to bypass these checks for automation workflows?

Thanks in advance!

**Comment by bewuethr** [comment]
I think that's how they currently work, see for example https://github.com/orgs/community/discussions/45484
